diff --git pylorax/__init__.py pylorax/__init__.py
index 04d13610..1c5deb7c 100644
--- pylorax/__init__.py
+++ pylorax/__init__.py
@@ -161,7 +161,7 @@ class Lorax(BaseLoraxClass):
             isfinal=False, workdir=None, outputdir=None, buildarch=None, volid=None,
             domacboot=False, doupgrade=True, remove_temp=False,
             installpkgs=None,
-            size=2,
+            size=10,
             add_templates=None,
             add_template_vars=None,
             add_arch_templates=None,
@@ -308,7 +308,8 @@ class Lorax(BaseLoraxClass):
             else:
                 logger.info("no BCJ filter for arch %s", self.arch.basearch)
         rb.create_runtime(joinpaths(installroot,runtime),
-                          compression=compression, compressargs=compressargs)
+                          compression=compression, compressargs=compressargs,
+                          size=size)
 
         logger.info("preparing to build output tree and boot images")
         treebuilder = TreeBuilder(product=self.product, arch=self.arch,
