diff --git pylorax/__init__.py pylorax/__init__.py
index 04d13610..1c5deb7c 100644
--- pylorax/__init__.py
+++ pylorax/__init__.py
@@ -161,7 +161,7 @@ class Lorax(BaseLoraxClass):
             isfinal=False, workdir=None, outputdir=None, buildarch=None, volid=None,
             domacboot=False, doupgrade=True, remove_temp=False,
             installpkgs=None,
-            size=2,
+            size=10,
             add_templates=None,
             add_template_vars=None,
             add_arch_templates=None,
